CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://book.async.rs/overview
 
 ## Unreleased
 
+## [10.0.0] - 2021-09-30
```

Comment:
Should probably mention here the perf improvements & refactorings that we landed

Reply:
Give me wording

Reply:
Integrate a variety of zk-SNARK proving related performance improvements. For details see https://github.com/filecoin-project/bellperson/issues/220